### DISCUSSION TITLE
Display reblog trail on dashboard?

### DISCUSSION THREAD

**Discussion by solosvejs** [AprilSylph/XKit-Rewritten#807] [Help Desk] [answered]
A few months ago Tumblr removed the ability to see the reblog trail on dashboard posts. By which I mean, additions to the post are now just separated by a horizontal line, and you cannot see the users who made those additions. This data is still there, though - it's visible if you go to reblog the post, and the mobile version of tumblr hasn't had this change (yet.... )

Is there some way to bring these back? I looked through all the options for the dashboard tweaks but couldn't find anything.

**Comment by marcustyphoon** [comment, marked as answer]
This is a behavior of Tumblr Savior. The dev refuses to turn it off by default and sometimes the setting gets stuck, but in theory you should (usually) be able to toggle it off in its settings without removing the extension.

**Reply by solosvejs** [reply]
Oh my god I never would've thought to look there. Thank you so much!

**Comment by AprilSylph** [comment]
https://github.com/bjornstar/Tumblr-Savior/issues/69
